README.md

```diff
@@ -13,7 +13,7 @@ LORIS (Longitudinal Online Research and Imaging System) is a self-hosted web app
 This Readme covers installation of LORIS version <b>23</b> on <b>Ubuntu</b>.
 ([CentOS Readme also available](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/CentOS/README.md)).
 
-Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/Setup) notes on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+Please consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information. Installation instructions and further information about LORIS for developers can be found on the [LORIS ReadTheDocs website](https://acesloris.readthedocs.io/en/latest/).
```

Comment:
The ReadTheDocs is just a web-friendly presentation of markdowns, and the Installation Instructions you mention there _are_ these Installation markdowns, no?  
This line suggests the RTD docs don't include the same install instruction content.

Reply:
The readthedocs does include the installation instructions but it also includes all of the documentation available under docs/wiki. Would it be better to just link to readthedocs and not link to `/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install` at all?

Reply:
Absolute links will break readthedocs, and readthedocs can't manager link to folders. It's fine if we don't want to include the main README there though.

Reply:
i'll leave the points about which links break to laetitia.  keep all the mentions, and just make clear the subset relationship between them - i.e. that this same content is _also_ readable on ReadTheDocs, which also includes more docs...